webtau-core/src/main/java/com/twosigma/webtau/reporter/ValueMatcherExpectationSteps.java

```diff
@@ -29,34 +31,35 @@
     public static <C, V> void shouldStep(C context, V value, StepReportOptions stepReportOptions, TokenizedMessage valueDescription, ValueMatcher valueMatcher) {
         executeStep(context, value, valueDescription, valueMatcher, false,
                 tokenizedMessage(action("expecting")),
-                () -> actual(value).should(valueMatcher), stepReportOptions);
+                runnable(() -> actual(value).should(valueMatcher)), stepReportOptions);
```

Comment:
Looks like all invocations of `executeStep` take what is actually a `Runnable`.  Therefore, I think we should either have `executeStep` take a `Runnable` and convert to a `Supplier` (rather than doing the conversion in each `executeStep` invocation site) or we create an overload of `TestStep.create` which takes a `Runnable` and converts it.

Reply:
Having looked through the rest of the review, I'd go for the `TestStep.create` overload as there are other classes that wrap a `Runnable` into a `Supplier` in exactly the same way.

Reply:
Makes sense. Thanks.